### PR TITLE
Update change tier page design

### PR DESCRIPTION
**Pull request by davidrapson**
This PR updates the change tier page to match the other tier pages (join / choose tier).

**The PR assumes that it is possible to perform a paid-to-paid upgrade which allows us to greatly simplify the logic here; so these changes are dependant on PR** https://github.com/guardian/membership-frontend/pull/181

**Large**

change tier large

**Medium**
change tier medum

**Small**
change tier small

### CODE REVIEW

File: frontend/assets/stylesheets/components/_icons.scss [hunk 1]
```diff
@@ -115,6 +115,13 @@
     }
 }
 
+.icon-holder--small {
+    width: 20px; height: 20px;
```
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_icons.scss]: Personally not super-keen on this style, find it a bit harder to scan/parse what the code is doing. Unless it was like...

    $iconWidth: 20px;
    width: $iconWidth; height: $iconWidth;

File: frontend/assets/stylesheets/components/_packages.scss [hunk 2]
```diff
@@ -362,6 +377,23 @@
     }
 }
 
+.package-promo--current {
+    background-color: $c-neutral5;
+}
+
+.package-promo--stretch {
+    @include mq(desktop) {
+        min-height: rem(gs-height(6.5));
```
Comment (mattandrews) [inline frontend/assets/stylesheets/components/_packages.scss]: Does this number actually have a link to the grid system or is it just co-incidence? Might as well just hardcode it if it's magic. If not, ignore me!

**Comment by mattandrews** [comment]
:+1: besides comments – needs merge though.

File: frontend/assets/stylesheets/components/_packages.scss (hunk 2 above)
Comment (davidrapson) [inline frontend/assets/stylesheets/components/_packages.scss]: Yeah, it's probably magic.

**Comment by davidrapson** [comment]
@jamesoram Mind giving this a once over?

File: frontend/app/views/tier/change.scala.html [hunk 3]
```diff
@@ -2,37 +2,76 @@
...
+                                        }
+                                    </ul>
+
+                                    @if(tier != currentTier) {
+                                        <a class="action"
+                                           href="@routes.TierController.upgrade(tier)"
+                                           data-metric-trigger="click"
+                                           data-metric-category="changeTier"
+                                           data-metric-action="@tier.toString.toLowerCase()"
+                                        >
```
Comment (jamesoram) [inline frontend/app/views/tier/change.scala.html]: could we add a hook here please? something like qa-confirm which is tier-independent.

Comment (davidrapson) [inline frontend/app/views/tier/change.scala.html]: This button is specific to a tier, are you sure you want it tier-independent?

**Comment by davidrapson** [comment]
I've added the Coming Soon check to the new templates so that we can launch this without needing paid-to-paid upgrades to be publicly available.

screen shot 2015-02-18 at 15 40 39

**Comment by davidrapson** [comment]
@jamesoram Added the extra qa hook to the downgrade page as requested

**Comment by jamesoram** [comment]
I noticed this weirdness

screen shot 2015-02-19 at 12 07 07 2

**Comment by davidrapson** [comment]
@jamesoram Thanks James, added the required check to the secondary button.

screen shot 2015-02-19 at 13 29 53

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**